packages/config/src/tokens/types/Token.ts

```diff
@@ -2,60 +2,66 @@
  * This file defines the original Tokens
  */
 interface Location {
-	parents: number;
-	interior: Record<string, unknown>;
+  parents: number;
+  interior: Record<string, unknown>;
 }
 
 interface TokenConstructorParams {
-	key: string;
-	symbol: string;
-	decimals: number;
-	location: any;
-	network: string;
+  key: string;
+  symbol: string;
+  decimals: number;
+  location: any;
+  network: string;
 }
 
 class Token {
-    public key: string;
-    public symbol: string;
-    public decimals: number;
-    public location: any; // Consider using a more specific type.
-    public network: string;
-  
-    constructor(params: TokenConstructorParams) {
-      this.key = params.key;
-      this.symbol = params.symbol;
-      this.decimals = params.decimals;
-      this.location = params.location;
-      this.network = params.network;
-    }
+  public key: string;
+
+  public symbol: string;
+
+  public decimals: number;
+
+  public location: any; // Consider using a more specific type.
+
+  public network: string;
+
+  constructor(params: TokenConstructorParams) {
+    this.key = params.key;
+    this.symbol = params.symbol;
+    this.decimals = params.decimals;
+    this.location = params.location;
+    this.network = params.network;
   }
+}
 
-  /** Below defines createToken for simple creation of Token */
+/** Below defines createToken for simple creation of Token */
 interface TokenParams {
-	key: string;
-	symbol: string;
-	decimals: number;
-	network: string;
-	parachainId: number;
-	palletInstance?: number;
+  key: string;
+  symbol: string;
+  decimals: number;
+  network: string;
+  parachainId: number;
+  x2Params?: any;
 }
 
 // Function to construct the 'location' object based on the parameters.
-function constructLocation(parachainId: number, palletInstance?: number): Location {
-	let interior = palletInstance ? { X2: [{ Parachain: parachainId }, { PalletInstance: palletInstance }] } : { X1: { Parachain: parachainId } };
+function constructLocation(parachainId: number, x2Params?: any): Location {
```

Comment:
Let’s rename the `x2Params` to `interior`.

Reply:
Fixed